apps-api/backoffice/src/modules/admin/poll/models.ts

```diff
@@ -1,30 +1,42 @@
-import { Poll, PollDetails } from '@pple-today/api-common/dtos'
-import { PollType } from '@pple-today/database/prisma'
+import {
+  AdminPollDetails,
+  ListPaginationQuery,
+  Poll,
+  PollDetails,
+} from '@pple-today/api-common/dtos'
+import { PollStatus, PollType } from '@pple-today/database/prisma'
 import { Static, t } from 'elysia'
 
 export const PollIdParams = t.Object({
   pollId: t.String({ description: 'The ID of the draft poll' }),
 })
 export type PollIdParams = Static<typeof PollIdParams>
 
-export const GetPollsQuery = t.Object({
-  type: t.Optional(
-    t.Union(
-      [
-        t.Literal('publish', { description: 'String `publish`' }),
-        t.Literal('draft', { description: 'String `draft`' }),
-      ],
-      {
-        description: 'Type of poll',
-      }
-    )
-  ),
-  limit: t.Optional(t.Number({ default: 10 })),
-  page: t.Optional(t.Number({ default: 1 })),
-})
+export const GetPollsQuery = ListPaginationQuery(
+  t.Object({
+    search: t.Optional(t.String({ description: 'The search query for the poll title' })),
+    status: t.Optional(
+      t.Array(
+        t.Union(
+          [
+            t.Literal('PUBLISHED', { description: 'String `publish`' }),
+            t.Literal('DRAFT', { description: 'String `draft`' }),
+            t.Literal('ARCHIVED', { description: 'String `archived`' }),
+          ],
+          {
+            description: 'Type of poll',
+          }
+        )
+      )
+    ),
```

Comment:
suggestion: you can use `t.Enum` if there are no reason for you to literate each value
```suggestion
    status: t.Optional(t.Array(t.Enum(PollStatus))),
```